front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -156,6 +156,7 @@ const ScenarioV2 = () => {
   }, [timetable, infra]);
 
   useEffect(() => {
+    console.log('timetable', timetable, trainIdUsedForProjection);
```

Comment:
Remnant from a debugging session? 😛

Reply:
The PR wasn't ready for review, sorry :)
We usually post on matrix to ask for review when ready (but you right, I should have put it on draft mode)

Reply:
I see, nevermind then!